### PR TITLE
MD-15, MIP-15, MG-0: Movement Glossary

### PR DESCRIPTION
**Pull request by l-monninger**
# Summary
[MD-15](https://github.com/movementlabsxyz/MIP/tree/l-monninger/movement-gloss/MD/md-15)
[MIP-15](https://github.com/movementlabsxyz/MIP/tree/l-monninger/movement-gloss/MIP/mip-15)
[MG-0](https://github.com/movementlabsxyz/MIP/tree/l-monninger/movement-gloss/MG/mg-0)

Introduces the Movement Glossary and Movement Gloss (MG) document type.

### CODE REVIEW

**Review by andyjsbell — CHANGES_REQUESTED** [review]
Looks great to me

Looks like you have some commits pulled in here from other branches possibly

**Comment by apenzk** [comment]
silly question. why is there a MIP15 and a MG15. why not just MIP15?

**Comment by apenzk** [comment]
what would still be useful is to have a complete document, for ease of overview. Possibly at the root directory. The MG could then come with an addition to a common file (glossary.md) to which the MG contributes. The term introduced in the glossary.md should reference the MD

**Comment by apenzk** [comment]
I suggest to move the MG template into the MG directory rather than the MIP directory

File: MD/md-15/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+**Justification**: A common format for glossary entries will make it easier to maintain and update the glossary.
+
+**Recommendations**: 
+- Create a new formatted document for glossary entries in the MIP repository. Call this document an MG, short for Movement Gloss.
+- Include sections for (1) term, (2) authorship, (3) concise definition, (4) catalog of references to definitions or usages of the term in other contexts, (5) example usages, and (6) errata.
+- Encourage formalism where applicable.
+
+### D2: Introduce Glossary Terms via an MIP
+
+**Justification**: Introducing glossary terms via an MIP will allow for a more formal process for introducing new terms.
```
Comment (apenzk) [inline MD/md-15/README.md:45]:
```suggestion
**Justification**: Introducing glossary terms via an MIP or MD will allow for a more formal process for introducing new terms.
```

Comment (apenzk) [reply MD/md-15/README.md:45]: There is no information at the root README. i have [opened a PR](https://github.com/movementlabsxyz/MIP/pull/22) for this

**Review by apenzk — CHANGES_REQUESTED** [review]
what would still be useful is to have a complete document, for ease of overview. Possibly at the root directory. The MG could then come with an addition to a common file (glossary.md) to which the MG contributes. The term introduced in the glossary.md should reference the MD.

Edit: i have added PR https://github.com/movementlabsxyz/MIP/pull/22 for this

**Pull request merged**